src/pages/costModels/components/rateForm/rateForm.tsx

```diff
@@ -73,6 +73,15 @@ const RateFormBase: React.FunctionComponent<RateFormProps> = ({
     });
     return label ? label : m;
   };
+  const getMeasurementDescription = (o, u) => {
+    // Match message descriptor or default to API string
+    const units = u.toLowerCase().replace('-', '_');
```

Comment:
Could you please add a comment here stating that `units` only works with Node, Cluster, and PVC? And note that `units` does not need to be translated in that case.

When the metric is CPU, Memory, or Storage, `units` equals something more like `core_hours` or `gb_hours` and must be translated.